TerrorConsole/Assets/_Main/Scripts/Objects/Switchs/OrderLeverController.cs

```diff
@@ -0,0 +1,55 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    public class OrderLeverController : MonoBehaviour
+    {
+        [SerializeField] private List<SwitchObject> levers;
+        
+        [SerializeField] private List<string> _correctOrder;
+        private List<string> _currentOrder = new List<string>();
+        
+        [SerializeField] private OrderSwitchLockedDoor _lockDoor;
+            
+        private void Start()
+        {
+            foreach (var lever in levers)
+            {
+                lever.OnActivated.AddListener(OnLeverActivated);
+                lever.OnDeactivated.AddListener(OnLeverDeactivated);
+            }
+        }
+
+        private void OnLeverActivated(string leverId)
+        {
+            _currentOrder.Add(leverId);
+            Debug.Log($"OnLeverActivated: {leverId}");
+            CheckForCorrectOrder();
+        }
+
+        private void OnLeverDeactivated(string leverId)
+        {
+            _currentOrder.Remove(leverId);
+            Debug.Log($"Lever {leverId} deactivated");
```

Comment:
Remove debugs, are useful when you are constructng the feature, not when it is complete, it will mess up to actual more useful logs.